09-ncvs-vignette.Rmd

```diff
@@ -530,9 +532,9 @@ We will discuss four estimate types as follows:
 
 $$ \hat{t}_D = \sum_{ijkl \in D} v_{ijkl}$$
 
-where $v_{ijkl}$ is the series-adjusted victimization weight for household $i$, respondent $j$, reporting period $k$, and victimization $l$, that is `WGTVICCY`.
+where $v_{ijkl}$ is the series-adjusted victimization weight for household $i$, respondent $j$, reporting period $k$, and victimization $l$, that is `WGTVICCY`. 
```

Comment:
these second parts are not indented like the rest of the text for each numbered item.  Can these be intended to line up?